packages/webdriver-utils/test/providers/automateProvider.test.js

```diff
@@ -29,21 +29,21 @@ describe('AutomateProvider', () => {
     });
 
     it('throws Error when called without initializing driver', async () => {
-      let automateProvider = new AutomateProvider('1234', 'https://localhost/command-executor', { platform: 'win' }, {}, {}, 'client', 'environment', {}, percyBuildInfo);
+      let automateProvider = new AutomateProvider('1234', 'https://localhost/command-executor', { platform: 'win' }, {}, {}, 'client', 'environment', percyBuildInfo);
```

Comment:
Do we want to change this?, the options ideally are the screenshot options, we should not remove it because
this will make `options` = `percyBuildInfo` where as `percyBuildInfo` = `undefined` unless we name make it named parameter. This gives an unclear code flow view.

Reply:
AutomateProvider constructor expects 8 parameters whereas we were sending 9, now percyBuildInfo is passed correctly at 8th position.
I checked locally using logging the value for percyBuildInfo was coming as `{}` whereas we were passing it but at wrong position.